cms_form/models/cms_search_form.py

```diff
@@ -128,15 +140,24 @@ def form_search_domain(self, search_values):
                 if not value:
                     continue
                 operator = 'in'
-            elif field['type'] in ('many2one', ) and not value:
-                # we need an existing ID here ( > 0)
-                continue
+            elif field['type'] in ('many2one', ):
+                value = int(value) if value.isdigit() else 0
```

Comment:
any chance to run into smth w/ spaces here? I.e., `'3 '.isdigit()` is `False`

Reply:
that's impossible, at this stage the value is clean ;)